chemie/events/views.py

```diff
@@ -554,15 +558,25 @@ def set_status_message(self, request, status, instance, event):
             )
 
         elif status == REGISTRATION_STATUS.INTERESTED:
-            messages.add_message(
-                request,
-                messages.INFO,
-                "Det er ikke åpent for ditt klassetrinn, "
-                "men vi har notert din interesse. Du blir påmeldt "
-                "automatisk og tilsendt en e-post dersom dette endres.",
-                extra_tags="Interessert",
-            )
-
+            if not event.allowed_group(instance.user):
+                messages.add_message(
+                    request,
+                    messages.INFO,
+                    "Denne påmeldingen er kun åpen for kohortsjefer."
+                    "Dersom du ønsker å bli påmeldt dette arrangementet, "
+                    "eller andre arrangementer under blotet, må du be din kohortsjef"
+                    "melde på kohorten din.",
+                    extra_tags="Ikke gruppemedlem",
```

Comment:
Same comment here, we should list which lines in which files are to be edited to a more general message, so I am just commenting here so we will remember =D